README.md

```diff
@@ -284,7 +349,7 @@ While this addon previously included a number of components that would aid in id
 
 ## Future Plans
 
-Now that your components and acceptance tests can self-audit, the next step
+Now that your components and application tests can self-audit, the next step
 going forward is to give helpful and meaningful feedback to developers. This
 means easily highlighting areas with violations and giving suggestions on how to
 fix and improve them. Additionally, work will be done to tackle Ember-specific
```

Comment:
Should probably remove this section as it talks about the visual audit component feature that was deprecated.

Reply:
Ya I was waiting for someone to notice this so I could delete it :)